core/config.py

```
@@ -10,6 +11,20 @@ class Settings(BaseSettings):
     database_url: str = "sqlite+aiosqlite:///./reconciliation.db"
     app_env: str = "development"
     log_level: str = "INFO"
+    app_base_url: str = "http://127.0.0.1:8000"
+    
+    # Authentication & JWT
+    jwt_secret_key: str = "your-super-secret-jwt-key-change-in-production"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "jwt_secret_key"

(S105)